marklogic-data-hub-central/ui/e2e/cypress/integration/curation/curate/mapping.spec.tsx

```diff
@@ -112,5 +121,9 @@ describe('Mapping', () => {
     browsePage.getTableViewSourceIcon().click();
     cy.contains('mappedOrderDate');
     cy.contains('categoryCode');
+
+    //Verifying the properties added by load and mapping custom hooks respectively
+    cy.contains('primaryKey');
+    cy.contains('uriFromCustomHook');
```

Comment:
can we pls add .should('be.visible') to all the four cy.contains()
I could be wrong, but I think, contains is a query and doesnt assert, so these steps might not fail if there is no mappedOrderDate or primaryKey in the DOM.

Reply:
cy.contains() will fail if the element is not found.